docs/platforms/kotlin-multiplatform/troubleshooting.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Troubleshooting
+description: "Troubleshoot and resolve edge cases."
+sidebar_order: 9000
+---
+
+If you need help solving issues with your Sentry Kotlin Multiplatform SDK, you can read the edge cases documented below. If you need additional help, you can [ask on GitHub](https://github.com/getsentry/sentry-kotlin-multiplatform/issues/new/choose). Customers on a paid plan may also contact support.
```

Comment:
```suggestion
This document covers common issues you may encounter when using the Sentry Kotlin Multiplatform SDK and provides steps to troubleshoot.

If you need additional help, you can [ask on GitHub](https://github.com/getsentry/sentry-kotlin-multiplatform/issues/new/choose). Customers on a paid plan may also contact support.
```
I feel like edge cases aren't really covered here

Reply:
Yeah, I think since for now we're just listing one specific problem, we should just talk about it directly as opposed to calling it an edge case. :)